File: 0771-jewels-and-stones/0771-jewels-and-stones.py
class Solution:
    def numJewelsInStones(self, jewels: str, stones: str) -> int:
        jewels = set(jewels)
        jewels_count: int = 0
        
        for stone in stones:
            if stone in jewels:
                jewels_count += 1

        return jewels_count
        

File: 0771-jewels-and-stones/0771-jewels-and-stones.java
class Solution {
    public int numJewelsInStones(String jewels, String stones) {
        Set<Character> jewelsSet = new HashSet<>();
        for (char j: jewels.toCharArray()) {
            jewelsSet.add(j);
        }

        int jewelsCount = 0;

        for (char s: stones.toCharArray()) {
            if (jewelsSet.contains(s)) {
                jewelsCount++;
            }
        }

        return jewelsCount;
    }
}